.github/workflows/ci.yaml

```diff
@@ -103,6 +103,14 @@ jobs:
         id: deadcode
         run: |
           make go/deadcode
+      - name: Deepcopy generation
```

Comment:
it kind of fits linting, but could we move this to a dedicated job that is separate from linting so it's clear what fails and we can extend it with other checks in the future

Reply:
Done